src/components/navbar/navbar.js

```diff
@@ -12,7 +12,7 @@ const Navbar = () => {
     localStorage.removeItem("token");
     localStorage.removeItem("userName");
     setAuthState({ isAuth: false, userInfo: null });
-    navigate("/login");
+    navigate(-1);
```

Comment:
i don't know about this  navigate(-1) can you just get some idea here so also learn some new

Reply:
I added -1 here because when viewer is not logged in and he wants to like a video then a alert will come that please login to like and then the user will login and after he login he will get on the same page he was trying to like the video.